components/global/main.tf

```diff
@@ -30,3 +30,24 @@ module "ctags" {
   builtFrom    = var.builtFrom
   expiresAfter = var.expiresAfter
 }
+
+module "frontdoor_classic" {
+  count  = var.env == "sbox" ? 1 : 0
+  source = "git::https://github.com/hmcts/terraform-module-frontdoor.git?ref=master"
+
+  common_tags                = module.ctags.common_tags
+  env                        = var.env
+  project                    = var.project
```

Comment:
I think you need to change the project name as there is already a front door classic with that name.

Reply:
aha - good spotted Tyler